tools/js-sdk-release-tools/test/changelogGenerator.test.ts

```diff
@@ -0,0 +1,16 @@
+import { expect, test } from 'vitest'
+import { extractExportAndGenerateChangelog } from '../src/changelog/extractMetaData'
+import path from 'path';
+
+test('HLC -> Modular: Rename', async () => {
+  const oldViewPath = path.join(__dirname, "testCases/operationGroups.old.api.md");
+  const newViewPath = path.join(__dirname, "testCases/operationGroups.new.api.md");
+  const changelog = await extractExportAndGenerateChangelog(oldViewPath, newViewPath);
+  
+  expect(changelog.addedOperationGroup.length).toBe(0);
+  expect(changelog.removedOperationGroup.length).toBe(0);
+  expect(changelog.interfaceParamDelete.length).toBe(1);
+  expect(changelog.interfaceParamDelete[0]).toBe('Interface DataProductsCatalogsOperations no longer has parameter listByResourceXXXGroup');
+  expect(changelog.interfaceParamAddRequired.length).toBe(1);
+  expect(changelog.interfaceParamAddRequired[0]).toBe('Interface DataProductsCatalogsOperations has a new required parameter listByResourceGroup');
```

Comment:
Should be `Added operation DataProductsCatalogsOperations.listByResourceGroup`?
https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/monitor/arm-monitor/CHANGELOG.md#L617
![image](https://github.com/Azure/azure-sdk-tools/assets/9943211/ed957497-6b7e-43c3-8563-92a497462d5c)